src/System.Windows.Forms/src/System/Windows/Forms/ButtonBase.ButtonBaseAccessibleObject.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Windows.Forms
+{
+    public partial class ButtonBase
+    {
+        public class ButtonBaseAccessibleObject : ControlAccessibleObject
+        {
+            public ButtonBaseAccessibleObject(Control owner) : base((owner as ButtonBase) ?? throw new ArgumentException(string.Format(SR.Argument_InvalidValueType, nameof(Owner), typeof(ButtonBase))))
+            {
+            }
+
+            private ButtonBase _owningButton => (ButtonBase)Owner;
```

Comment:
[naming] This is a property not a field
```suggestion
            private ButtonBase OwningButton => (ButtonBase)Owner;
```

Reply:
Thanks, fixed